doc/usecases/polyculture/map_specific_favorite_groups.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Map Specific Favorite Groups
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user change a set of favorite plant groups per map.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged in to the app.
+  - The plants layer is selected.
+  - At least one plant is placed
+- **Main success scenario:**
+  - The user adds or removes a plant group from her list of favorites.
+- **Alternative scenario:**
```

Comment:
Currently, there is no alternative scenario listed. I have a few suggestions:

- The user wants to reorder their list of favorite plant groups
- The user wants to reset their list of favorite plant groups to the default settings

You can dismiss them if you think that is just bloat.

Reply:
Good idea, there definitely should be a removal of favorites.